### ISSUE TITLE
iOS transmit button label is the same whether you're transmitting or muted.

### ISSUE THREAD

**Issue by Simon818** [BearWare/TeamTalk5#775]
In the latest iOS beta as of today (and not in the one before it), the transmit button is read by VoiceOver as "toggle to enable / disable transmission". This used to be the hint for the button, but now there s seemingly no hint and the old "transmit" and "stop transmitting" accessibility labels are gone. The button looks the same now whether you're transmitting or not, so a VoiceOver user won't know the state of the button until they press it.

**Comment by Oreonan** [comment]
I've replaced accessibility hints by accessibility labels. I'll try more 
later tomorrow to see why label is not changing when button state changed.

**Comment by Simon818** [comment]
I think because the hint is the same regardless of whether the button is enabled or disabled. It might be better to give it a shorter label, like "transmit", and then change the state so that VoiceOver says "selected" before it when the button is enabled. This is what a lot of voice chatting apps do for their mute, video, and other buttons.

**Comment by Simon818** [comment]
Now that I look around the app more, I can see why you did this. A lot of the accessibility labels are really helpful, and there was information in hints that arguably should have been in the labels. I would still advocate for the "transmit" label though. I also notice that the tabs across the bottom now read their index twice. For instance, when I touch the messages tab, VoiceOver says "messages tab, 2 of 3, 2 of 3." Not sure if this is because of your changes; if so, maybe we should make a separate issue to talk about ideas for them, because I like where you're going with it.

**Comment by Oreonan** [comment]
For now I've restored accessibility hint, like previously, for tx button. I'll see if I can find a way to change state of a button with an accessibility string. For your other question, I don't think it link to my change.
Please let me know if others accessibility hint should be restored instead of accessibility label.

**Comment by Oreonan** [comment]
AccessibilityHint is back in latest beta, and I've added an AccessibilityValue for tx button.

**Comment by ChrisDuffley** [comment]
That 2 of 3 twice issue is not happening for me as far as I’m aware. I just confirmed that what I just stated as correct.
